website/docs/r/elastic_cloud_elasticsearch.html.markdown

```diff
@@ -39,7 +39,10 @@ The following arguments are supported:
 
 * `resource_group_name` - (Required) The name of the Resource Group where the Elasticsearch resource should exist. Changing this forces a new Elasticsearch to be created.
 
-* `sku_name` - (Required) Specifies the name of the SKU for this Elasticsearch. Changing this forces a new Elasticsearch to be created.
+* `sku_name` - (Required) Specifies the name of the SKU for this Elasticsearch. Changing this forces a new Elasticsearch to be created.\
```

Comment:
```suggestion
* `sku_name` - (Required) Specifies the name of the SKU for this Elasticsearch. Changing this forces a new Elasticsearch to be created.
```

Reply:
Done in https://github.com/hashicorp/terraform-provider-azurerm/pull/23863/commits/a4a09265230796da306f2a546028c7afd8bcb100.
![image](https://github.com/hashicorp/terraform-provider-azurerm/assets/85922721/de0b7e99-c5b8-469f-b496-d1c719db6b77)